scripts/script.js

```diff
@@ -0,0 +1,77 @@
+var answers = [];
+
+
+
+// questions constructor
+
+var Questions = function(flavor, question){
```

Comment:
Remember that the object remains with a singular name because you will create instances of that object